### DISCUSSION TITLE
STORAGE_CLASS in AWS

### DISCUSSION THREAD

**Discussion by mariomare22** [cupcakearmy/autorestic#361] [Q&A]
Hello!
Is there a way to upload to AWS S3 and specify the storage class when uploading? to DEEP_ARCHIVE for example.

uploading to STANDARD and then lifecycling to DEEP_ARCHIVE creates overhead that causes higher monthly cost.
Thank you!

**Comment by LM1LC3N7** [comment]
Same question. I know restic can use an option for that: `-o s3.storage-class`. Does autorestic support that?
https://restic.readthedocs.io/en/latest/080_examples.html

**Reply by LM1LC3N7** [reply]
Ok, found the solution thanks to https://github.com/cupcakearmy/autorestic/discussions/341#discussioncomment-7746462.

```
option:
    - s3.storage-class=GLACIER_IR
```

In my case I wanted to use "Glacier Instant Retrieval", as this seems to be the cheapest options for incremental backups like restic.
